src/Http/Session.php

```diff
@@ -226,7 +226,9 @@ public function regenerateId()
 			if (headers_sent($file, $line)) {
 				throw new Nette\InvalidStateException('Cannot regenerate session ID after HTTP headers have been sent' . ($file ? " (output started at $file:$line)." : '.'));
 			}
-			session_regenerate_id(TRUE);
+			if (session_id() !== '') {
+				session_regenerate_id(TRUE);
+			}
 			session_write_close();
```

Comment:
It seems working, but this situation is kinda rare, so if another error will show up, i will fix it here.